libs/blocks/fragment/fragment.js

```diff
@@ -63,6 +63,13 @@ export default async function init(a) {
   let relHref = localizeLink(a.href);
   let inline = false;
 
+  if (a.parentElement && a.parentElement.nodeName === 'P') {
+    const children = a.parentElement.childNodes;
+    const div = document.createElement('div');
```

Comment:
Would there be situations where the `p` element has some additional classes/attributes? If so, this logic would strip them

Reply:
This is a great question and should be tested. My guess is there is no case since it's added by the franklin pipeline as wrapping element - but better to make sure now that we port this to a more prominent chunk of code.

Reply:
Good catch! Made the change to address this use case too 👍